<launch>

  <node name="krb_tf_listener" pkg="knowrob_beliefstate" type="krb_tf_listener" output="screen"/>
  <param name="initial_package" type="string" value="knowrob_beliefstate" />
  <node name="json_prolog" pkg="json_prolog" type="json_prolog_node" output="screen"/>

</launch>
